lib/inc/cpp-pcp-client/connector/connection.hpp

```diff
@@ -58,7 +62,9 @@ static const std::string DEFAULT_CLOSE_REASON { "Closed by client" };
 
 // Configuration of the WebSocket transport layer
 
-using WS_Client_Type = websocketpp::client<ws_config>;
```

Comment:
Could we use this for non-Windows platforms?

Reply:
We had a slack conversation about looking in to that after this release gets out if we cant fix it for windows. https://puppet.slack.com/archives/CGJ0GTF4Y/p1638486827029700 for now we were thinking of just prioritizing a quick fix.

Reply:
I think we probably can. We're reverting this for the puppet agent release that's supposed to ship in just a few days, so I want to wait until after that to experiment with selectively turning it on.